packages/expo-updates/android/src/androidTest/java/expo/modules/updates/loader/EmbeddedLoaderTest.kt

```diff
@@ -59,6 +59,7 @@ class EmbeddedLoaderTest {
     )
 
     every { mockLoaderFiles.readEmbeddedManifest(any(), any()) } returns manifest
+    every { mockLoaderFiles.copyAssetAndGetHash(any(), any(), any()) } answers { callOriginal() } // test for exception cases
```

Comment:
do we care about analogous tests for iOS?

Reply:
i'm going with manual tests for now since we don't have analogous unit tests for this class on iOS and it would take a fair bit of work to set up, and i want to get this fix out asap.